config/controller/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: public.ecr.aws/aws-controllers-k8s/route53-controller
-  newTag: 0.0.5
+  newTag: 0.0.0-non-release-version
```

Comment:
It seems that you did not fetch the tags for the `route53-controller` repo when generating this. Could you please do a `git fetch --all` and regenerate?

Reply:
done